types/lib/associations/belongs-to-many.d.ts

```diff
@@ -22,7 +22,13 @@ export interface ThroughOptions {
   /**
```

Comment:
Please add tests for these typing changes

Reply:
I have added a test, please let me know if the test I have added is inadequate or in the wrong place.